### PR TITLE
Updating Hugo theme recommendations and related code/links

### PR DESCRIPTION
**Pull request by apreshill**
Hi @yihui,

Due to recent updates in how Hugo themes work and the new way that the Hugo Academic theme is set up, I wanted to update our theme recommendations in the book. We can always add that back in as a recommended theme for blogdown users if the workflow improves.

Alison

### CODE REVIEW

File: docs/01-introduction.Rmd [hunk 1]
```diff
@@ -304,8 +304,8 @@ blogdown::new_site(theme = 'lxndrblz/anatole')
 To save you some time, we list a few themes below that match our taste:
 
 - Simple/minimal themes: [XMin,](https://github.com/yihui/hugo-xmin),
-[Cupper,](https://github.com/zwbetz-gh/cupper-hugo-theme),
-[Tanka,](https://github.com/road2stat/hugo-tanka) [simple-a,](https://github.com/AlexFinn/simple-a) and [ghostwriter.](https://github.com/jbub/ghostwriter)
```
Comment (yihui) [inline docs/01-introduction.Rmd]: @nanxstats Do you want to submit your theme to https://github.com/gohugoio/hugoThemes?

**Review by yihui — APPROVED** [review]
Looks good. I'll merge once I hear back from @nanxstats. Thank you!

File: docs/01-introduction.Rmd (hunk 1 above)
Comment (nanxstats) [reply docs/01-introduction.Rmd]: @yihui -yes, i will take a look at the submission process over the weekend.

Comment (yihui) [reply docs/01-introduction.Rmd]: Sounds good. I'll put your theme back into the book then. Thanks!

Comment (nanxstats) [reply docs/01-introduction.Rmd]: Hey @yihui -just a heads up that I found the theme actually requires a larger update to fully work under the current Hugo version, and especially, Goldmark... I should be able to fix them next week after taking care of some work stuff this week. Thanks again for the patience and my apologies for the inconvenience.

Comment (yihui) [reply docs/01-introduction.Rmd]: No worries and no hurry at all. The main goal of this PR was to remove hugo-academic, which has been achieved, so please take your time.

**Comment by nanxstats** [comment]
I have rewritten the theme entirely to make sure it runs with the latest Hugo and [submitted](https://github.com/gohugoio/hugoThemes/issues/945) it to the official theme gallery. I will report back once it's accepted there.

The submission checks are a bit overwhelming, but hopefully future Hugo versions won't break things too frequently.

**Comment by yihui** [comment]
The checks do look overwhelming! I'm not sure if I'd have the nerve the submit my next theme there :)

**Comment by nanxstats** [comment]
Hey @yihui it is finally on the official theme gallery, I created a PR to update the link 👉 https://github.com/rstudio/blogdown/pull/551

**Pull request merged**